tests/lib/tools/journal-state

```diff
@@ -1,19 +1,25 @@
 #!/bin/bash
 
-# shellcheck source=tests/lib/state.sh
-. "$TESTSLIB/state.sh"
-
 # This file contains all the cursors used for the different tests
-JOURNALCTL_CURSOR_FILE="$RUNTIME_STATE_PATH"/journalctl_cursor
+JOURNALCTL_CURSOR_FILE="$SPREAD_PATH"/tests/runtime-state/journalctl_cursor
+
+show_help() {
+    echo "usage: journal-state start-new-log"
+    echo "usage: journal-state check-ready"
```

Comment:
Out of the whole set the only name that doesn't immediately make sense to me is `check-ready`. Perhaps `is-working` or something similar. This function is only called in one place so it can have a longer name.

Reply:
Already moved to check-log-started, thanks for the review